### DISCUSSION TITLE
The development environment works properly, but not when deployed to iis

### DISCUSSION THREAD

**Discussion by zhang1446802857** [hardkoded/puppeteer-sharp#2455] [Q&A]
### Description
When I developed using VS, I used PupperSharp. Its functionality is very powerful and meets my requirements, and I did not find any issues during debugging because it can be used normally. But when I deployed the project to the IIS server, I found an issue: 'Failed to create connection'. I have searched a lot of information and made modifications from many places, including the version of Google Chrome, which seems to have nothing to do with these, so I don't know what to do now.

### Complete minimal example reproducing the issue

E.g.

```csharp
 //下载凭证
            await new BrowserFetcher().DownloadAsync(BrowserFetcher.DefaultChromiumRevision);
            //创建一个浏览器实例
            var browser = await Puppeteer.LaunchAsync(new LaunchOptions
            {
                ExecutablePath = @"C:\Users\Administrator\Desktop\projblog\.local-chromium\Win64-1069273\chrome-win\chrome.exe",
                //无头模式
                Headless = true
            });
            //创建一个新页面
            var page = await browser.NewPageAsync();
            //导航加载指定Url
            await page.GoToAsync(url);


```

### Versions

* Which version of PuppeteerSharp are you using?
* latest
* Which .NET runtime and version are you targeting? E.g. .NET framework 4.6.1 or .NET Core 2.0.
* netcore 6

**Comment by James-Zhangpp** [comment]
Are you and I encountering the same problem and seeking a solution together?

**Comment by James-Zhangpp** [comment]
I have encountered the same problem as you and are seeking solutions together

**Comment by James-Zhangpp** [comment]
I have encountered the same problem as you and are seeking solutions together

**Comment by zhang1446802857** [comment]
> Are you and I encountering the same problem and seeking a solution together?

有点久了。我记得我应该换成了[Selenium]

**Comment by kblok** [comment]
IIS needs a custom transport factory. See https://stackoverflow.com/a/62429184/2373249
